src/components/merchantPage/MenuInfo.jsx

```diff
@@ -66,12 +61,12 @@ const MenuInfo = ({
                         icon={faMapMarkerAlt}
                         className="w-4 h-4 mr-2"
                     />
-                    {locationName}
+                    {address}
                 </a>
             </div>
             <div className="mt-2 flex items-center">
                 <FontAwesomeIcon icon={faPhone} className="w-4 h-4 mr-2" />
-                {telNumber}
+                {phoneNumber}
             </div>
 
             <div className="mt-2">
```

Comment:
The business hours in `merchantData`  are not being mapped to the display of business hours